src/compas_rhino/geometry/surface.py

```diff
@@ -31,13 +31,20 @@ def from_selection(cls):
         guid = compas_rhino.select_surface()
         return cls.from_guid(guid)
 
-    def to_compas(self, cls=None):
+    def to_compas(self, cls=None, facefilter=None, cleanup=True):
         """Convert the surface b-rep loops to a COMPAS mesh.
 
         Parameters
         ----------
         cls : :class:`compas.datastructures.Mesh`, optional
             The type of COMPAS mesh.
+        facefilter : callable, optional
+            A filter for selection which Brep faces to include.
+            Default is ``None``.
```

Comment:
This might lead to some confusion.  The default behavior is to include all faces, right?  This could be read to say that no faces are included.